src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -434,6 +435,13 @@ export default {
 			}
 			this.$emit('submit', event)
 		},
+		/**
+		 * Esc key pressed
+		 * @param {Event} event the keydown event
+		 */
+		onEsc(event) {
+			this.$emit('esc', event)
```

Comment:
Also, as a regular note, you should not change the event type. Here this is not an `esc` event, this is a  `keydown`.
Staying as close to html/js as possible is easier to understand for other devs

Reply:
I initially called it `keydown.esc`, but it was not working. I believe It was conflicting with the vue listener on the parent.

Reply:
because the event is a `keydown`, not a `keydown.esc`